server/src/main/java/io/crate/execution/engine/pipeline/TableFunctionApplier.java

```diff
@@ -57,7 +58,11 @@ public TableFunctionApplier(List<Input<Iterable<Row>>> tableFunctions,
             expressions.get(i).setNextRow(row);
         }
         mapIncomingValuesToOutgoingCells();
-        List<Iterator<Row>> iterators = Lists2.map(tableFunctions, x -> x.value().iterator());
+        List<Iterator<Row>> iterators = tableFunctions.stream()
+            .map(Input::value)
+            .filter(Objects::nonNull)
+            .map(Iterable::iterator)
+            .collect(Collectors.toList());
```

Comment:
I don't this this should be changed here. Rather the functions shouldn't return null elements.